### PR TITLE
Update Chat.tsx

### PR DESCRIPTION
**Pull request by Norton71**
### Contribution Checklist

- [x] I have built and tested the code locally and in a deployed app
- [x] For frontend changes, I have pulled the latest code from main, built the frontend, and committed all static files.
- [x] This is a change for all users of this app. No code or asset is specific to my use case or my organization.
- [x] I didn't break any existing functionality :smile:

### CODE REVIEW

**Comment by Norton71** [comment]
Good

File: frontend/src/pages/chat/Chat.tsx [hunk 1]
```diff
@@ -865,7 +865,7 @@ const Chat = () => {
               </Stack>
               <QuestionInput
                 clearOnSend
-                placeholder="Type a new question..."
+                placeholder="Type a new question please..."
```
Comment (Norton71) [inline frontend/src/pages/chat/Chat.tsx:868]: OK

Comment (Norton71) [inline frontend/src/pages/chat/Chat.tsx:868]: ok

**Comment by abhahn** [comment]
Hi @Norton71 , this PR appears to contain customizations for your specific application, which we are unable to merge since it will affect all users of the repository.

I'm going to close this PR, but if you have a common feature or bugfix which can benefit all users please feel free to open another one in the future.

**Pull request closed without merging**